templates/taxi/car_detail.html

```diff
@@ -1,6 +1,13 @@
 {% extends "base.html" %}
 
 {% block content %}
+  <h3>
```

Comment:
The `<h3>` tag is not properly opened. Ensure that the opening tag is correctly formatted to avoid rendering issues.